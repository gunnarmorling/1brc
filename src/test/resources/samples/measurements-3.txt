Bosaso;5.0
Bosaso;20.0
Bosaso;-5.0
Bosaso;-15.0
Petropavlovsk-Kamchatsky;9.5
Petropavlovsk-Kamchatsky;-9.5
