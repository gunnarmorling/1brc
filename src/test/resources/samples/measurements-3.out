{Bosaso=-15.0/1.3/20.0, Petropavlovsk-Kamchatsky=-9.5/0.0/9.5}
